src/index.js

```diff
@@ -81,14 +82,25 @@ export default class ScrollPlugin extends Plugin {
         window.history.scrollRestoration = 'auto';
     }
 
+    getOffset = () => {
+        switch (typeof this.options.offset) {
+            case 'number':
+                return this.options.offset;
+            case 'function':
+                return this.options.offset();
+            default:
+                return Number(this.options.offset);
```

Comment:
Since we're scrolling "in pixels", would it make sense to use `parseInt` instead, to make sure it is an integer, eg. whole pixels?

Reply:
Sure, no harm done by being specific.